src/commands/compile_raw.rs

```diff
@@ -51,7 +51,7 @@ fn serialize_cairo_program_debug_info(debug_info: &CairoProgramDebugInfo) -> Vec
         .iter()
         .map(|statement_debug_info| {
             (
-                statement_debug_info.code_offset,
+                statement_debug_info.start_offset,
```

Comment:
Make sure to confirm it by running `snforge` tests (at least some of them) on Scarb `2.7.0-rc.0`

Reply:
When run using `statement_info.end_offset`, `snforge` tests fail